composer.json

```diff
@@ -19,6 +19,9 @@
         "symfony/browser-kit": "~2|~3",
         "symfony/process": "~2|~3"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.0"
```

Comment:
Yeah it works too. Now it only downloads php-cs-fixer when needed, so I like it, but I can change it.